about.html

```diff
@@ -0,0 +1,161 @@
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Program</a></li>
+            <li><button type="button">sports Campaign</button></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section class="ban">
+        <h1 class="margin-left">the sports summit  program</h1>
+        <div class="ban-summary move white-bg">
+          <p>
+            Like all good things in this world,
+             tournaments cost money to create.
+              Before spending any money on the event, 
+              a budgetary plan and goal needs to be set.
+               Nevertheless, creating a budget for a youth football
+                tournament can be difficult, 
+            especially for those who are hosting for the first time.
+           
+          </p>
+        </div>
+        <p class="place side">
+          Please contact per Email for any further questions about sports summit
+          program 2021 <br />
+          <span>rochidzwasports@gmail.com</span>
+        </p>
+      </section>
+
+      <section class="about">
+        <h3>The countdown sports program 2022 logo</h3>
+        <hr class="line" />
+        <p>
+         
+        </p>
+        <div class="about-logo">
+          <img src="images/Capture.png" alt="" />
+        </div>
+      </section>
+
+      <section class="past-events">
+        <h3>See the past sports summit s</h3>
+        <hr class="line" />
+        <p class="event-summary">
+          Take a look at last two sports summit s which took place virtually
+        </p>
+        <div class="event-list">
+          <div id="event-0">
+            <div class="bg-color">
+              <span>2021</span> <br />
+              sports summit 
+            </div>
+          </div>
+          <div id="event-1">
+            <div class="bg-color">
+              <span>2022</span> <br />
+              sports summit 
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="sponser off">
+            <h3>Partner</h3>
+            <hr class="line" />
+            <ul class="sponser-list">
+              <li class="image"><img src="images/Coca-Cola-Logo-StartupTalky--1-.jpg" alt="" /></li>
+              <li><img src="images/Budweiser-Logo-StartupTalky.jpg" alt="" /></li>
+              <li><img src="images/Qatar-Airways-Logo-StartupTalky.jpg" alt="" /></li>
+              <li class="image">
+                <img src="images/Vivo-Logo-StartupTalky--1-.jpg" class="stretch" alt="" />
+              </li>
+              <li class="image">
+                <img src="images/Wanda-Group-Logo-StartupTalky.jpg" class="stretch" alt="" />
+              </li>
+            </ul>
+          </section>
+        </main>
+        <footer>
+          <div class="footer color-w">
+            <img src="images/Capture.png" alt="" />
+            <p>2021 sports, all rights reserved</p>
+          </div>
```

Comment:
- [ ] Kindly make the footer on the about page dark as seen in the original design

![image](https://user-images.githubusercontent.com/4247740/164861808-dce35031-19c7-49ac-96f6-50af1bd0844c.png)